src/rebuild/srv.c

```diff
@@ -712,7 +665,7 @@ rebuild_global_pool_tracker_create(struct ds_pool *pool, uint32_t ver,
 	D_INIT_LIST_HEAD(&rgt->rgt_list);
 
 	node_nr = pool_map_node_nr(pool->sp_map);
-	array_size = roundup(node_nr, DAOS_BITS_SIZE) / DAOS_BITS_SIZE;
```

Comment:
can you remove the DAOS_BITS_SIZE macro in common.h to avoid confuse?
there is no daos_first_unset_bit usage and it is incorrect now, probably also can be removed.
can in other patch